crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -105,7 +103,7 @@ enum LogState {
     /// create the initial segment configuration.
     Provisioning {
         log_id: LogId,
-        provider_kind: ProviderKind,
+        cluster_configuration: Arc<ClusterConfiguration>,
```

Comment:
What happens if the `ClusterConfiguration` changes while we are trying to provision (e.g. waiting for 3 nodes to become available and then the `ClusterConfiguration` changes to a replication factor of `1`)?

Reply:
Yeah, i already removed that. was a bad decision